src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -408,6 +413,17 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
       }
     }
     this.deviceInfoCache = devices;
+    this.setDeviceIdGroupIdMap();
+  }
+
+  private setDeviceIdGroupIdMap(): void {
+    for (const device of this.deviceInfoCache) {
+      if (device.groupId) {
```

Comment:
- nit: It can be in just one line without this `if` statement.
    ```
    this.deviceIdGroupIdMap[device.kind].set(device.deviceId, device.groupId || '');
    ```
- nit: Since you can simplify it, you can just put it into `updateDeviceInfoCacheFromBrowser()`.